docs/Components.md

```diff
@@ -0,0 +1,67 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.9.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Calico Components
+
+## Components of Basic Calico Container Deployments
+
+All basic Calico deployments require the following components:
+
+ - **`calico/node`**: Docker image that runs the underlying calico processes (see 
+   ***Anatomy of calico/node*** below). The `calico/node` image must be deployed 
+   on each host in the network.
+
+ - **`calicoctl`**: The command line tool responsible for configuring all of the 
+   networking components within Calico, including endpoint addresses, security 
+   policies, and BGP peering.
+
+ - **Orchestrator (Docker, Kubernetes, etc)**: Manages container creation/deletion 
+   and runs the `calico/node` Docker image as a container.
+
+ - [**etcd**](https://github.com/coreos/etcd): Datastore used by Calico to store 
+   endpoint, bgp, and policy data that can be accessed by all of the hosts.  
+   See the [etcd Data Model page](./etcdStructure.md) for details on how Calico 
+   data is stored in etcd. 
+
+## Anatomy of calico/node
+
+`calico/node` can be regarded as a helper container that bundles together the 
+various components required for Calico networking.  The image utilizes the 
+following processes:
```

Comment:
Several parts of this (Felix, BIRD as BGP client/vRR) are covered in more detail in the [OpenStack Calico docs](http://docs.projectcalico.org/en/stable/architecture.html).

Is it worth adding a link to those docs? The extra detail is useful, in addition to the summaries already written.

(There may be a bigger question here about having Calico docs split across two places, but this probably isn't the place for that discussion.)